consensus/src/simplex/actors/batcher/actor.rs

```diff
@@ -116,19 +118,23 @@ impl<
                 match self.notarizes[index as usize] {
                     Some(ref previous) => {
                         if previous != &notarize {
-                            let activity = ConflictingNotarize::new(previous.clone(), notarize);
-                            self.reporter
-                                .report(Activity::ConflictingNotarize(activity))
-                                .await;
+                            if self.attributable_scheme {
```

Comment:
This will impact our approach to implementing #1921 (it would otherwise be easy to just listen to this stream) 🤔